python/rmm/rmm/__init__.py

```diff
@@ -58,6 +60,12 @@ def __getattr__(name):
     if name == "_lib":
         import importlib
 
+        warnings.warn(
```

Comment:
The warnings only warn once. Should they warn on every import?

Reply:
No, I think a single warning is fine.